go.mod

```
@@ -1,6 +1,6 @@
 module github.com/NBISweden/sda-cli
 
-go 1.21
+go 1.22
```

```suggestion
go 1.22.3
```

You need the patch version as well for CodeQL to work